controllers/profiles/followUserV3.js

```diff
@@ -84,9 +90,16 @@ module.exports = async (req, res) => {
   try {
     await BetterSocialCore.fcmToken.sendMultiDeviceNotification(
       req?.userId,
-      username_follower,
+      user?.username,
+      user_id_followed,
+      targetUser?.username
+    );
+
+    Getstream.chat.sendFollowSystemMessage(
+      req?.userId,
+      user?.username,
       user_id_followed,
-      username_followed
+      targetUser?.username
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The changes made here are good for improving code readability and maintainability. By using `user?.username` and `targetUser?.username`, you're making it clear that these values are derived from the `user` and `targetUser` objects, which makes the code easier to understand. Also, this change reduces the reliance on the request body for important data, which can improve security by reducing the risk of injection attacks.


<!-- This is an auto-generated comment by OSS CodeRabbit -->